servers/Azure.Mcp.Server/src/Azure.Mcp.Server.csproj

```diff
@@ -7,6 +7,7 @@
     <ReadmeUrl>https://github.com/Microsoft/mcp/blob/main/servers/Azure.Mcp.Server#readme</ReadmeUrl>
     <ReadmePath>$(RepoRoot)/servers/Azure.Mcp.Server/README.md</ReadmePath>
     <PackageIcon>$(MSBuildThisFileDirectory)../azureicon.png</PackageIcon>
+    <McpRepositoryName>com.microsoft/azure</McpRepositoryName>
```

Comment:
Do we require any approvals to use this name on NuGet?

Reply:
This is only used in MCP repository.